utils/rowgroup/rowgroup.h

```diff
@@ -1370,7 +1370,17 @@ inline void Row::setLongDoubleField(const long double& val, uint32_t colIndex)
 {
     uint8_t* p = &data[offsets[colIndex]];
     *((long double*)p) = val;
-    if (sizeof(long double) == 16)
+    
+    
+    int64_t d = std::numeric_limits<long double>::digits;  // Number of base 2 digits (64 for 80 bit fpu)
```

Comment:
If we have a global bool we can calculate it once on startup and fill the global to leverage it later. This should save us 2 mov instr + 1 cmp perRow::setLongDoubleField call.